backend/btrixcloud/orgs.py

```diff
@@ -501,6 +517,8 @@ async def rename_org(
         org: Organization = Depends(org_owner_dep),
     ):
         org.name = rename.name
+        if rename.slug:
+            org.slug = rename.slug
```

Comment:
Do we want to auto-generate the slug whenever only the name changes?
```suggestion
            org.slug = rename.slug
       else:
            org.slug = slug_from_name(rename.name)
```